editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -158,10 +158,7 @@ bool VisualShaderEditor::_is_available(int p_mode) {
 		p_mode = temp_mode;
 	}
 
-	if (p_mode != -1 && ((p_mode & current_mode) == 0)) {
-		return false;
-	}
-	return true;
+	return (p_mode != -1 && ((p_mode & current_mode) == 0));
```

Comment:
Lacks a `!`, no?

It's also a bit hard to read, maybe `(p_mode == -1 || (p_mode & current_mode) != 0)` would be better? (Still hard to read though, but maybe slightly better :D).